performance-metrics/src/performance_tests.rs

```diff
@@ -429,7 +429,7 @@ mod tests {
 }
        "#;
         assert_eq!(
-            parse_iperf3_output(output.as_bytes(), true).unwrap(),
+            parse_iperf3_output(output.as_bytes(), true, true).unwrap(),
```

Comment:
Can you please extend the unit tests to cover the PPS parsing?

Reply:
Sure. Done.